KBH/6Day/예산.js

```diff
@@ -0,0 +1,18 @@
+function solution(d, budget) {
+    const sorted = d.toSorted((a, b) => a - b);
+    
+    let sum = 0;
+
+    let count = 0;
```

Comment:
```suggestion
    const sorted = d.toSorted((a, b) => a - b);
    let sum = 0;
    let count = 0;
```
@KimBYH ㅎㅎㅎ 모든 변수를 줄바꿈을 할 필요는 없습니다 ㅎㅎㅎ 문맥 상의 분리만 해도 충분합니다.

Reply:
배열을 정렬했다는 의미가 담긴 sortedArray가 좋은것 같습니다!
수정하겠습니다!